From 56ed7395164c46866f96c510398876e70687c47a Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Sat, 28 Sep 2019 11:32:50 +0200
Subject: [PATCH] build: Remove tests for _readdir() and __readdir()

These date back to pre-git e19196110cc9d29c3fb330737507adc364f3bd9a in 1998.

More of these should be removed but the _readdir() test caused a timeout
on an ARM builder in Debian. It might just be a fluke but the tests are
pointless regardless.

See also 16eb80058ca4a34df96388c26736590247b6a3f4 "build: Remove tests for
getdents() and getdirentries()"

Signed-off-by: Mathieu Parent <math.parent@gmail.com>
Forwarded: https://gitlab.com/samba-team/samba/merge_requests/821
---
 source3/wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source3/wscript b/source3/wscript
index e22cb16f3fb..6e759e8caec 100644
--- a/source3/wscript
+++ b/source3/wscript
@@ -403,7 +403,7 @@ _open __open _opendir __opendir
 pathconf poll
 posix_memalign pread _pread __pread
 pwrite
-_read __read _readdir __readdir
+_read __read
 _seekdir __seekdir
 select setenv setgidx setgroups setlocale setluid
 setmntent setpgid setpriv setsid setuidx
-- 
2.20.1

